percy/har-mock-config.yml

```diff
@@ -0,0 +1,125 @@
+branding: HAR test
+title: test environment
+homeTimezone: America/Chicago
+
+# Default OTP API
+api:
+  host: http://localhost:9999
+  # host: http://localhost:8001 # For testing against a local OTP instance
```

Comment:
Remove this line

Reply:
Good call! Removed in 46a7627572c613a1abb4e1e13e1a481c6e7a3245